### PR TITLE
Add JDTLS with Lombok

### PR DESCRIPTION
**Pull request by alanjjenkins**
Hey @kabouzeid ,

Can you take another look at this please? I have been using it the last few weeks without any issues but please check it over to see if you can find anything else that you think needs fixing (I cannot see anything wrong with it as it is though).

Thanks,

Alan Jenkins

### CODE REVIEW

**Comment by kabouzeid** [comment]
Thanks for your effort. I probably don't have time today, but will look at this soon :)

File: lua/lspinstall/servers/java.lua [hunk 1]
```diff
@@ -0,0 +1,80 @@
+local config = require'lspconfig'.jdtls.document_config
+require'lspconfig/configs'.jdtls = nil -- important, immediately unset the loaded config again
+
+config.default_config.cmd[1] = {
+  "./jdtls.sh",
+  vim.fn.stdpath('data') .. '/lspinstall/java/workspace/' ..
```
Comment (kabouzeid) [inline lua/lspinstall/servers/java.lua:6]: can also be relative

`'./workspace/' ..`

File: lua/lspinstall/servers/java.lua [hunk 2]
```diff
@@ -0,0 +1,80 @@
+local config = require'lspconfig'.jdtls.document_config
+require'lspconfig/configs'.jdtls = nil -- important, immediately unset the loaded config again
+
+config.default_config.cmd[1] = {
```
Comment (kabouzeid) [inline lua/lspinstall/servers/java.lua:4]: This should be `cmd`, are you *sure* `cmd[1]` worked here? If so, something is off.

File: lua/lspinstall/servers/java.lua [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  vim.fn.stdpath('data') .. '/lspinstall/java/workspace/' ..
+  string.gsub(
+    vim.fn.fnamemodify(vim.fn.getcwd(), ':p'),
+    "/",
+    "-"
+  )
+}
+
+return vim.tbl_extend('error', config, {
+  install_script = [[
```
Comment (kabouzeid) [inline lua/lspinstall/servers/java.lua:15]: Can this be run multiple times without errors? This is so we can update the server by running `:LspInstall java` again.

**Comment by shaunsingh** [comment]
Just would like to say it installs and runs fine.

**Comment by rafamadriz** [comment]
Just tested and it also works for me.

**Comment by Gaarco** [comment]
Doesn't work for me, it installs but when I open Neovim I get this error:
```E5113: Error while calling lua chunk: ...ite/pack/packer/start/nvim-lspinstall/lua/lspinstall.lua:102: bad argument #1 to 'match_str' (string expected, got table)```

**Pull request merged**